SEImplementation/src/lib/Plugin/FlexibleModelFitting/FlexibleModelFittingTask.cpp

```diff
@@ -299,8 +300,12 @@ void FlexibleModelFittingTask::computeProperties(SourceGroupInterface& group) co
   }
 
   // Model fitting
+
+  // FIXME we can no longer specify different settings with LeastSquareEngineManager!!
```

Comment:
Indeed. Each engine has a different set of configuration parameters. May be worth exposing them as plugins do, so users can tweak the tolerances if they wish. The problem is ModelFitting "lives" kind of decoupled from this plugin mechanism, but I think it should be relatively straight-forward to move the engines themselves to a plugin.